control-plane/roles/monitoring/templates/alertmanager/alertmanager-rules.yaml

```diff
@@ -0,0 +1,41 @@
+{% raw %}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    release: prometheus
+    role: alert-rules
+  name: metal-api.rules
+  namespace: monitoring
```

Comment:
This line can be removed as it's already specified by the `k8s` module using the appropriate namespace variable. Occurs multiple times in this PR's diff, so I do not comment all occurrences.